### ISSUE TITLE
[Feat] - Add Issue templates

### ISSUE THREAD

**Issue by abhishekHegde2000** [vishalverma9572/QuizMaster#67]
**Description:**
Create issue templates to standardize the format and ensure all necessary information is provided for different types of issues. These templates will be added to the `.github/ISSUE_TEMPLATE/` directory.

**Tasks:**
1. **Create Directory:**
   - If it doesn't already exist, create the `.github/ISSUE_TEMPLATE/` directory.

2. **Create Bug Report Template:**
   - Create a `bug_report.yml` file in the `.github/ISSUE_TEMPLATE/` directory with the necessary template content for bug reports.

3. **Create Feature Request Template:**
   - Create a `feature_request.yml` file in the `.github/ISSUE_TEMPLATE/` directory with the necessary template content for feature requests.

**Files to Modify:**
- Create `.github/ISSUE_TEMPLATE/bug_report.yml`
- Create `.github/ISSUE_TEMPLATE/feature_request.yml`

**Benefits:**
- Standardizes the format for submitting bug reports and feature requests, improving clarity and completeness.
- Facilitates easier and more consistent issue tracking and management within the project.

**Comment by abhishekHegde2000** [comment]
@Sanchitagarwal7 Can you please assign this task to me under hactoberfest ?

**Comment by vishalverma9572** [comment]
@abhishekHegde2000  , assigned! By the way, I’m Vishal. Please let me know what exactly you're planning to improve first before starting.

**Comment by abhishekHegde2000** [comment]
Hi Vishal,

While everyone wants to contribute to the project, there is currently no systematic way to create features or report a bug. To streamline this process, we should create an issue template. Once you click on "new issue," everyone will be prompted with two options: feature or bug. It's important that everyone follows these rules.

**Comment by vishalverma9572** [comment]
@abhishekHegde2000 , thank you for your wonderful contribution! This repository really needed an issue template, and you nailed it. Could you please create a similar template for PRs? It should require images for frontend UI changes and Postman response images for backend updates. Thanks again!

**Comment by vishalverma9572** [comment]
@abhishekHegde2000 ,  I am creating a new issue for the PR template task and assigning it to you. Thanks again for your help!

**Comment by vishalverma9572** [comment]
@abhishekHegde2000  checkout #77
